system-addon/lib/ActivityStream.jsm

```diff
@@ -27,21 +30,74 @@ XPCOMUtils.defineLazyModuleGetter(this, "TelemetryFeed",
 XPCOMUtils.defineLazyModuleGetter(this, "TopSitesFeed",
   "resource://activity-stream/lib/TopSitesFeed.jsm");
 
-const feeds = {
-  // When you add a feed here:
-  // 1. The key in this object should directly refer to a pref, not including the
-  //    prefix (so "feeds.newtabinit" refers to the
-  //    "browser.newtabpage.activity-stream.feeds.newtabinit" pref)
-  // 2. The value should be a function that returns a feed.
+// Preferences configuration
+const DEFAULT_PREFS = [
```

Comment:
Not sure if this is the right place to list non-default add-on prefs.. although the only one should be "browser.newtabpage.activity-stream.enabled" right now? But if there are others, this list could easily diverge…